models/sipnet/inst/register.SIPNET.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<SIPNET>
```

Comment:
Not exactly -- the name in the pulldown and in the met processing code are all Ameriflux. What's different is that the file format name is not the name as the source, which I don't think will be unique. Could easily have a name tag at a high level that's different from the name tag within the format record.

Agree that I can't recall why we set up the met registry files like this.